components/centraldashboard/karma.conf.js

```diff
@@ -5,7 +5,7 @@ webpackConfig.module.rules.push({
     enforce: 'post',
     test: /\.js$/,
     use: {
-        loader: 'istanbul-instrumenter-loader',
+        loader: 'babel-plugin-istanbul', // Updating loader since istanbul not used 7 years
```

Comment:
I avoid comments like this, just because it won't add any clarity to the code base long term.
But that's maybe just me.

Reply:
Keeping it in exception since it might trigger an error for the testing later on